source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/utils/wrappers/skrl.py

```diff
@@ -62,6 +62,18 @@ def reward_shaper(rewards, timestep, timesteps):
         return reward_shaper
 
     def update_dict(d):
+        # import statements according to the ML framework
+        if ml_framework.startswith("torch"):
```

Comment:
why startswith instead of `==`?

Reply:
Since there is `torch`, `jax` and `jax-numpy`... and JAX is compared using `startswith` to encapsulate both cases (`jax` and `jax-numpy`), I used the same style for `torch`:

```
if args_cli.ml_framework.startswith("torch"):
    pass
elif args_cli.ml_framework.startswith("jax"):
```